### ISSUE TITLE
Arrest report page

### ISSUE THREAD

**Issue by jxckUK** [SnailyCAD/snaily-cadv4#692]
### Describe the feature

Can we get a page to view all arrest reports for LEO supervisors so they can review and a simple option to archive/accept/sign any reports to move them off the page and into an archived section?

**Comment by Dev-CasperTheGhost** [comment]
You can already view citizen's arrest reports, tickets, warnings. What you're requesting, is a new page where arrest reports *need* to be accepted by supervisors before they're actually "accepted" to the arrestee?

**Comment by jxckUK** [comment]
> You can already view citizen's arrest reports, tickets, warnings. What you're requesting, is a new page where arrest reports _need_ to be accepted by supervisors before they're actually "accepted" to the arrestee?

Yes you can but you cannot view *all* citizen arrest reports in one view. The _need_ should be an optional feature, but yes that would also be a very nice addition. You can then see what supervisor approved the arrest report.

**Comment by Dev-CasperTheGhost** [comment]
This would only account for Arrest reports, not tickets and written warnings?
